pom.xml

```diff
@@ -1006,6 +1010,10 @@
               <goal>integration-test</goal>
               <goal>verify</goal>
             </goals>
+            <configuration>
+              <excludeGroups>${accumulo.integrationTestsExcludedGroups}</excludeGroups>
+              <groups>${accumulo.integrationTestsGroups}</groups>
+            </configuration>
```

Comment:
It's very likely we will have more categories soon, including those for surefire, so it'd be a small bit of future-proofing work, so we don't have to completely redo the docs.